automation/gce-deploy/site.conf

```diff
@@ -53,26 +43,30 @@ LDAPCacheTTL ${LDAP_CACHE_TTL}
     RewriteCond %{HTTP:CONNECTION} ^Upgrade$ [NC]
     RewriteRule .* ws://app:8080%{REQUEST_URI} [P]
 
-    <Location ${PROXY_PATH}>
-        # For /notebooks URLs, Leo handles setting CORS headers itself
-        Header unset Access-Control-Allow-Origin "expr=%{REQUEST_URI} !~ m#^/notebooks#"
-        Header always set Access-Control-Allow-Origin "*" "expr=%{REQUEST_URI} !~ m#^/notebooks#"
-        Header unset Access-Control-Allow-Headers "expr=%{REQUEST_URI} !~ m#^/notebooks#"
-        Header always set Access-Control-Allow-Headers "authorization,content-type,accept,origin" "expr=%{REQUEST_URI} !~ m#^/notebooks#"
-        Header unset Access-Control-Allow-Methods "expr=%{REQUEST_URI} !~ m#^/notebooks#"
-        Header always set Access-Control-Allow-Methods "GET,POST,PUT,PATCH,DELETE,OPTIONS,HEAD" "expr=%{REQUEST_URI} !~ m#^/notebooks#"
-        Header unset Access-Control-Max-Age "expr=%{REQUEST_URI} !~ m#^/notebooks#"
-        Header always set Access-Control-Max-Age 1728000 "expr=%{REQUEST_URI} !~ m#^/notebooks#"
-
-        RewriteEngine On
-        RewriteCond %{REQUEST_METHOD} OPTIONS
-        RewriteCond %{REQUEST_URI} !^/notebooks
-        RewriteRule ^(.*)$ $1 [R=204,L]
+    <Location "/">
```

Comment:
I haven't looked at this in detail, I'll leave that to someone who is already familiar with what this config was being used for.